build/build/src/aws/ecr/runtime.rs

```diff
@@ -42,18 +72,66 @@ mod tests {
     /// The engine must be already built.
     #[tokio::test]
     #[ignore]
-    async fn test_name() -> Result {
+    async fn test_runtime() -> Result {
         setup_logging().ok();
-        let tag = "test_runtime_image";
+        let tag = "runtime:0.0.0-dev";
         info!("Current directory: {}", ide_ci::env::current_dir()?.display());
         let root = deduce_repository_path()?;
         let root = root.absolutize()?;
         info!("Repository root: {}", root.display());
         let engine_package = generated::EnginePackage::new_root(
-            root.join("built-distribution/enso-engine-2023.1.1-dev-linux-amd64/enso-2023.1.1-dev"),
+            root.join("built-distribution/enso-engine-0.0.0-dev-linux-amd64/enso-0.0.0-dev"),
+        );
+        let dockerfile =
+            generated::RepoRootToolsCiDockerEngine::new_root(root.join("tools/ci/docker/engine"));
+        let id = build_runtime_image(&dockerfile, &engine_package, tag.to_string()).await?;
+        info!("Built image: {}", id);
+        Ok(())
+    }
+
+    /// Convenience test that builds the polyglot Ydoc image.
+    ///
+    /// The Ydoc native image must be already built.
+    #[tokio::test]
+    #[ignore]
+    async fn test_ydoc_polyglot() -> Result {
```

Comment:
Why is this called `test_`? What does it test?

Reply:
> /// Convenience test that builds the polyglot Ydoc image.

It builds the polyglot Ydoc docker image

Reply:
Why it is called `test_` then?